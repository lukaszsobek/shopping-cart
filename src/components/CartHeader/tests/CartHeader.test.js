import { CartHeader } from "../";

describe("CartHeader",() => {
    const props = {
        clearAllItems: () => null,
        openAddItemModal: () => null
    }
    const testHeader = new CartHeader(props);
    it("renders",() => {
        expect(testHeader.render()).toMatchSnapshot();
    });
});