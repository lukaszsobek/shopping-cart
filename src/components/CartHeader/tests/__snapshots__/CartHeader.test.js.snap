// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartHeader renders 1`] = `
<div
  className="top-bar"
>
  <div
    className="app-title"
  >
    Shopping cart
  </div>
  <div
    className="button-container"
  >
    <Button
      action={[Function]}
      className="clear-cart-button"
      label="Clear Cart"
    />
    <Button
      action={[Function]}
      className="add-item-button"
      label="Add Product"
    />
  </div>
</div>
`;
